jetty-core/jetty-util/src/main/java/org/eclipse/jetty/util/ssl/SslContextFactory.java

```diff
@@ -157,9 +157,9 @@ public abstract class SslContextFactory extends ContainerLifeCycle implements Du
     private Resource _trustStoreResource;
     private String _trustStoreProvider;
     private String _trustStoreType;
-    private Password _keyStorePassword;
-    private Password _keyManagerPassword;
-    private Password _trustStorePassword;
+    private Credential _keyStorePassword;
+    private Credential _keyManagerPassword;
+    private Credential _trustStorePassword;
```

Comment:
May as well rename private fields:
```suggestion
    private Credential _keyStoreCredential;
    private Credential _keyManagerCredential;
    private Credential _trustStoreCredential;
```